pages/initiatives/gsoc/gsoc2023ideas.md

```diff
@@ -523,3 +523,74 @@ Please use the repositories' issue tracker, GitHub discussions, and don't forget
 
 - [Viyat Bhalodia](mailto:viyat.bhalodia@owasp.org)
 - [Abraham Aranguran](mailto:Abraham.Aranguren@owasp.org)
+
```

Comment:
Add yourselves to the index or breadcrumbs at the top of the page?

Reply:
Ah good point!
I've overlooked that, thank you :)